### PR TITLE
Log ClientId in Telemetry

### PR DESCRIPTION
**Pull request by WardenGnaw**
ClientId will be useful to know when repoducing crashes on specific clients.

### CODE REVIEW

File: src/OpenDebugAD7/AD7DebugSession.cs [hunk 1]
```diff
@@ -1890,14 +1894,14 @@ protected override void HandleEvaluateRequestAsync(IRequestResponder<EvaluateArg
...
-            {
-                dapEvalFlags |= DAPEvalFlags.CLIPBOARD_CONTEXT;
-            }
-
             IDebugProperty2 property;
             if (expressionObject is IDebugExpressionDAP expressionDapObject)
             {
+                if (context == EvaluateArguments.ContextValue.Clipboard)
+                {
+                    dapEvalFlags |= DAPEvalFlags.CLIPBOARD_CONTEXT;
```
Comment (pieandcakes) [inline src/OpenDebugAD7/AD7DebugSession.cs:1902]: Scope dapEvalFlags to the context of the `if` statement here. its not necessary to have it scoped outside of this as it isn't used except for the call below.

**Comment by pieandcakes** [comment]
Two things: 
* what are the possible values for the AdapterID?
* don't forget that you will need to classify it before it is available.

**Comment by WardenGnaw** [comment]
> what are the possible values for the ClientID?
Looking to capture vscode and visualstudio.

**Comment by andrewcrawley** [comment]
> 
> 
> > what are the possible values for the ClientID?
> 
> Looking to capture vscode and visualstudio.

Might also be worth capturing "liveshare-server-host".

**Comment by gregg-miskelly** [comment]
We currently already know what IDE MIEngine is running in because each IDE logs telemetry to a different Kusto table. Is there a reason we need this?

**Comment by WardenGnaw** [comment]
I was looking at the telemetry and it seems like we wont log which IDE is being used if it causes a crash in the OpenDebugAD7 side?

Also there may be clientId specific things to do in AD7DebugSession so I wanted to have access to that variable.

**Comment by gregg-miskelly** [comment]
But each IDE is ultimately responsible for logging all telemetry. So we never need to log which IDE we are talking to -- its implied.

**Pull request closed without merging**